src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -242,7 +257,7 @@ private IceAdapterApi newIceAdapterProxy() {
             log.warn("Ignoring call to ICE adapter as we are not connected: {}({})", method.getName(), argList);
             return null;
           }
-          log.debug("Calling {}({})", method.getName(), argList);
+          log.warn("Calling {}({})", method.getName(), argList);
```

Comment:
That's nothing to `warn` about

Reply:
I think this should be a warning, this is not usual behaviour?

Reply:
Calling a method on the ICE adapter is a usual behavior

Reply:
oh fu, euhm, yes, that is, sry, I mistook this for being part of the not connected case above.